build.gradle.kts

```diff
@@ -2,11 +2,11 @@ import org.jetbrains.ValidatePublications
 import org.jetbrains.publicationChannels
 
 plugins {
-    org.jetbrains.conventions.base
-    id("org.jetbrains.dokka")
-    id("io.github.gradle-nexus.publish-plugin")
-
-    id("org.jetbrains.kotlinx.binary-compatibility-validator")
+    id("org.jetbrains.conventions.base")
+    id("org.jetbrains.dokka") version "1.8.10"
+    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
+    id("com.gradle.plugin-publish") version "0.20.0"
+    id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.12.1"
```

Comment:
It looks like the versions are duplicated here and in `build-logic/build.gradle.kts` now? Is it possible to have them in one place only?

Reply:
Seems can't, these plugins are used in `rootProject`.